src/zulqer/finalDirection/finalDirection.ts

```diff
@@ -0,0 +1,101 @@
+enum validDirections {
+  NORTH,
+  EAST,
+  SOUTH,
+  WEST
+}
+enum validMoves {
+  STRAIGHT,
+  RIGHT,
+  LEFT,
+  BACK
+}
+const MOVE_REWARD = {
+  STRAIGHT: 0,
+  RIGHT: 1,
+  LEFT: -1,
+  BACK: 2
+};
+const numberOfDirection = 4;
+
+const SAMPLE_VALID_INPUT: Array<validMoves> = [
+  validMoves.STRAIGHT,
```

Comment:
The test data should be kept in the test file, the main code should be free from any test data. The function being exported shall be tested from anywhere with custom inputs.